packages/editor/src/server/index.tsx

```diff
@@ -32,6 +32,7 @@ async function asyncMain() {
   const app = express()
 
   app.use('/assets', express.static(joinPath(__dirname, '../../assets'), {index: false}))
+  app.use('/static', express.static(path.resolve(__dirname, '../../static/')))
```

Comment:
I suggest you use `joinPath` here as well.

Reply:
ok and `{index: false}` as well?

Reply:
@0815pascal what does `{index: false}` do?

Reply:
> By default this module will send "index.html" files in response to a request on a directory. To disable this set false or to supply a new index pass a string or an array in preferred order.


Not sure if I understand this correctly. Does it mean, if I try for example to access the `/assets` directory from my browser, that it will create by default an `/assets/index.html` which shows all the content of `asset`-folder?

Reply:
👍🏾  so `{index: false}`seems like a good option.

Reply:
Ok, will do that!